finality-aleph/src/lib.rs

```diff
@@ -48,8 +48,8 @@ pub fn peers_set_config() -> sc_network::config::NonDefaultSetConfig {
         set_config: sc_network::config::SetConfig {
             // This seems to be a way to configure the AlephBFT network to have a prespecified set of nodes or at least
             // set suitable limits on the number of nodes we should connect to.
-            in_peers: 0,
-            out_peers: 0,
+            in_peers: 50,
+            out_peers: 50,
```

Comment:
I honestly never saw these numbers as having any effect whatsoever when trying it. 
`NetworkConfiguration` struct seems to have yet another `SetConfig`:
https://github.com/paritytech/substrate/blob/nil/client/network/src/config.rs#L419
that carries it's own setting for in and out peers (25 and 75 by default). 

Looking through the substrate code there seems to be a few places where they iterate over the `extra_sets.reserved_nodes` but not in/out peers... 
I have a suspicion someone just re-used the struct to carry the vector of reserved_nodes...

Maybe just call `SetConfig::default()` as extra_sets in the upstream and remove this code entirely?

Reply:
Done.